src/cdp-proxy/reactNativeCDPProxy.ts

```diff
@@ -20,15 +19,23 @@ export class ReactNativeCDPProxy {
     private port: number;
     private debuggerTarget: Connection;
     private applicationTarget: Connection;
-    private outputChannelLogger: OutputChannelLogger;
-    private logLevel: LogLevel;
+    private logger: OutputChannelLogger;
 
-    constructor(port: number, hostAddress: string, logLevel: LogLevel) {
+    private readonly PROXY_LOG_TAGS = {
+        DEBUGGER_COMMAND: "Command Debugger To Target",
+        APPLICATION_COMMAND: "Command Target To Debugger",
+        DEBUGGER_REPLY: "Reply From Debugger To Target",
+        APPLICATION_REPLY: "Reply From Target To Debugger",
+    };
+
+    constructor(port: number, hostAddress: string, logsEnabled: boolean) {
         this.port = port;
         this.hostAddress = hostAddress;
-        this.logLevel = logLevel;
+        this.logger = OutputChannelLogger.getChannel("RN CDP Proxy", true, false, true);
```

Comment:
```suggestion
        this.logger = OutputChannelLogger.getChannel("React Native Chrome Proxy", true, false, true);
```

Reply:
Not sure if we need to show this abbreviation to customers

Reply:
The name `React Native Chrome Proxy` could be clearer to understand, so I think the previous one should be replaced.